vent/controller/control_module.py

```diff
@@ -501,67 +499,6 @@ def __start_new_breathcycle(self):
             self.dl.flush_logfile()        # If we kept records, flush the data from the previous breath cycle
             self.dl.rotation_newfile()     # And Check whether we run out of space for the logger
 
-    def _STATECONTROL_update(self, dt):
```

Comment:
this stuff is all gone. As per our discussion: we only support and test the PID system